zerver/tests/test_decorators.py

```diff
@@ -310,15 +313,18 @@ def my_webhook_raises_exception_unsupported_event(
         # Test when content_type is application/json and request.body
         # is valid JSON; exception raised in the webhook function
         # should be re-raised
-        with mock.patch('zerver.decorator.webhook_logger.exception') as mock_exception:
+
+        # using assertLogs!
+        with self.assertLogs('zulip.zerver.webhooks', level='INFO') as mock_exception:
```

Comment:
This comment doesn’t add anything (just remove it), and `mock_exception` is a misleading variable name after this change.

Reply:
Thank you for this comment- please refer to our updated changes to this unit test